tests/pecan2.tests.xml

```diff
@@ -4,13 +4,10 @@
 
  <browndog>
    <!--
-   <url>http://dap.ncsa.illinois.edu:8184/convert/</url>
-   <username>browndog.user</username>
-   <password>GritGin3</password>
+     token expire after 24 hours
```

Comment:
Does the BrownDog package allow to generate a token easily? If so can you put the command here?

Reply:
done